rpcs3/rpcs3qt/progress_dialog.cpp

```diff
@@ -0,0 +1,62 @@
+#include "progress_dialog.h"
+
+progress_dialog::progress_dialog(int min, int max, QWidget* parent, Qt::WindowFlags flags) : QProgressDialog(parent, flags)
+{
+#ifdef _WIN32
+	m_tb_button = std::make_unique<QWinTaskbarButton>();
+	m_tb_button->setWindow(parent->windowHandle());
+	m_tb_progress = m_tb_button->progress();
+	m_tb_progress->setRange(min, max);
+	m_tb_progress->setVisible(true);
+#elif HAVE_QTDBUS
+	m_max = max;
+	UpdateProgress(0);
+	progressValue = new int(0);
```

Comment:
Is there some specific reason for using `int*` for progressValue?

Reply:
no idea, I just copied over hcorions code

Reply:
@hcorion any reason it can't be just a regular `int`?

Reply:
Nope, I can't remember why I did that. Maybe I was trying to make it similar to windows code?

Reply:
@Megamouse looks like it can be safely made into a regular `int `then.